baseservice/iodevices/file_system/file_system_device_manager_base.py

```diff
@@ -12,68 +11,71 @@
 
 class FileSystemDeviceManagerBase:
     """
-    this manager is used to create FileSystem IO Devices
+    this manager is used to create IO Devices over a shared file system
     """
     DEFAULT_QUEUES_SUB_DIR = 'QUEUES'
+    DEFAULT_BOOKKEEPING_SUB_DIR = 'BOOKKEEPING'
+    DEFAULT_TMPDIR_SUB_DIR = 'TMP'
```

Comment:
is there really any need to put these strings into consts? Are they ever used for anything else besides the constructor?

Reply:
no. but I like grouping them in the start of the module, for readability...